docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1523,6 +1523,94 @@ Note: If you use PostgreSQL database as IBM FHIR Server data store or the JavaBa
 
 For more information about Liberty JavaBatch configuration, please refer to [IBM WebSphere Liberty Java Batch White paper](https://www-03.ibm.com/support/techdocs/atsmastr.nsf/webindex/wp102544).
 
+### 4.10.1 *Path* and *Virtual Host* Bucket Access 
+
+For BulkData storage types of `ibm-cos` and `aws-s3`, the IBM FHIR Server supports two styles of accessing the `s3` bucket - virtual host and path.  In the IBM FHIR Server, `path` is the default access. [Link](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html)
+
+With path style access, the objects in a bucket are accessed using the pattern - `https://s3.region.host-name.com/bucket-name/object-key`. To configure path style access, one needs to configure the fhir-server-config.json. 
+
+There are three critical elements in the configuration to configure path style: 
+
+|Configuration|Details|
+|-------------|-------|
+|`endpointInternal`|the direct API provider for the S3 API|
+|`endpointExternal`|the fhiruser/client facing API for the S3 API|
+|`accessType`|"path", the default access type|
+
+Example of `path` based access: 
+
+``` json
+"bulkdata": {
+    ...
+    "storageProviders": {
+        "default" : {
+            "type": "aws-s3",
+            "bucketName": "bucket-name",
+            "location": "us",
+            "endpointInternal": "https://s3.region.host-name.com",
+            "endpointExternal": "https://s3.region.host-name.com",
+            "auth" : {
+                "type": "hmac",
+                "accessKeyId": "example",
+                "secretAccessKey": "example-password"
+            },
+            "enableParquet": false,
+            "disableBaseUrlValidation": true,
+            "exportPublic": true,
+            "disableOperationOutcomes": true,
+            "duplicationCheck": false,
+            "validateResources": false,
+            "create": false,
+            "presigned": true,
+            "accessType": "path"
+        }
+    }
+}
+```
+
+With virtual host style access, the objects in a bucket are accessed using the pattern - `https://bucket-name.s3.region.host-name.com/object-key`. To configure virtual host style access, one needs to configure the API. 
+
+There are three critical elements in the configuration to configure virtual host style: 
+
+|Configuration|Details|
+|-------------|-------|
+|`endpointInternal`|the direct API provider for the S3 API, and not the virtual host, the underlying S3 libraries generate the virtual host url|
+|`endpointExternal`|the fhiruser/client facing API for the S3 API, this is the virtual host url|
```

Comment:
question:  is endpointExternal used at all in our processing or do we just append the created resource to it at the end?

Reply:
endpointExternal is used at the end with bulkdata-status only

Reply:
it's just in the download, I've clarified in an updated phrase for endpointExternal